### ISSUE TITLE
Graduate scroll to text fragment 

### ISSUE THREAD

**Issue by marcoscaceres** [WICG/admin#176]
We should see if we can help scroll to text fragment get graduated over to the WHATWG. 

I think it's only waiting on the PR to get merged into HTML, but we should see if we can help in some way. 

https://github.com/WICG/scroll-to-text-fragment/issues/195
